pkg/prometheus/promcfg.go

```diff
@@ -488,6 +488,10 @@ func generateAlertmanagerConfig(version semver.Version, am v1.AlertmanagerEndpoi
 		cfg = append(cfg, k8sSDWithNamespaces([]string{am.Namespace}))
 	}
 
+	if am.BearerTokenFile != "" {
+		cfg = append(cfg, yaml.MapItem{Key: "bearer_token_file", Value: am.BearerTokenFile})
```

Comment:
is this configuration key already exist in the Alertmanager ?

Reply:
Yep see [alertmanager_config](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#<alertmanager_config>)